ntoskrnl/include/internal/ke.h

```diff
@@ -155,7 +155,6 @@ extern VOID __cdecl KiInterruptTemplate(VOID);
 
 /* MACROS *************************************************************************/
 
-#define AFFINITY_MASK(ProcessorIndex) ((KAFFINITY)1 << (ProcessorIndex))
```

Comment:
You may also remove the define that is in the HAL:
```
hal/halx86/include/smp.h	
  10 #define AFFINITY_MASK(Id) ((KAFFINITY)1 << (Id))
```

Reply:
done